### DISCUSSION TITLE
[Bug] Ollama not available anymore

### DISCUSSION THREAD

**Discussion by thomclae33** [lobehub/lobehub#1614] [Self Hosting] [answered]
### 💻 Operating System

Windows

### 📦 Environment

Docker

### 🌐 Browser

Other

### 🐛 Bug Description

Ollama is not available anymore in the LLM's (settings). Could this be as of the update [v0.133.5](https://github.com/lobehub/lobe-chat/releases/tag/v0.133.5)?

[a comment by an automated account is not shown]

**Comment by zkwolf** [comment]
use ollama > 0.1.24 && openapi setting

**Comment by biociao** [comment, marked as answer]
Check [the doc](https://chat-docs.lobehub.com/en/usage/features/local-llm#how-to-start-a-conversation-with-local-llm)  and make sure `OLLAMA_PROXY_URL` was set when startup the docker:
```
docker run -d -p 3210:3210 -e OLLAMA_PROXY_URL=http://host.docker.internal:11434/v1 lobehub/lobe-chat
```
